firefox-ios/Shared/Strings.swift

```diff
@@ -3569,6 +3569,11 @@ extension String {
             tableName: "TabsTray",
             value: "Close Old Tabs…",
             comment: "Text for a button in the tabs tray used to open another menu to close older tabs.")
+        public static let TabTrayCloseTabsTitle = MZLocalizedString(
+            key: "TabTrayCloseOneTabTitle.v145",
+            tableName: "TabsTray",
+            value: "Close Tabs: %@",
```

Comment:
```suggestion
            value: "Closed Tabs: %@",
```
Also nit in comment: (...) used *to* delete current active tabs.